feathr_project/feathr/spark_provider/_synapse_submission.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, synapse_dev_url: str, pool_name: str, datalake_dir: str, exec
         # use the same credential for authentication to avoid further login.
         self._api = _SynapseJobRunner(
             synapse_dev_url, pool_name, executor_size=executor_size, executors=executors, credential=self.credential)
+        # Suggest to use an empty datalake directory for a new spark job to avoid errors.
```

Comment:
Right now this note is only visible to dev, possible to make user be aware of this?

Reply:
Moved this note to the configuration doc.